compiler/ballerina-lang/src/main/java/io/ballerina/projects/util/ProjectUtils.java

```diff
@@ -143,6 +144,16 @@ && validateUnderscoresOfName(packageName)
                 && validateInitialNumericsOfName(packageName);
     }
 
+    /**
+     * Validates the package name.
+     *
+     * @param packageName The package name.
+     * @return True if valid package name, else false.
+     */
+    public static boolean validatePackageNameForNewCommand(String packageName) {
```

Comment:
Can we make this a part of ProjectUtils.validatePackageName without having to introduce a new API?